benchmarks/asv_numpy.conf.json

```diff
@@ -102,7 +105,7 @@
 
     // The directory (relative to the current directory) that benchmarks are
     // stored in.  If not provided, defaults to "benchmarks"
-    "benchmark_dir": "benchmarks",
+    "benchmark_dir": "benchmarks/numpy_",
```

Comment:
Why underscore?

Reply:
because then when you import numpy, "it"(guess who) thinks this is numpy ..

Reply:
can you rename them into something else, because this is confusing.

For example `numpy_benchmark`